libs/designsystem/src/lib/components/checkbox/checkbox.component.scss

```diff
@@ -76,21 +81,25 @@ $spacing-to-label: map-get($checkbox-radio-spacing, 'to-label');
 
   &.has-label {
     display: block;
+    line-height: line-height('n');
+    white-space: pre-line;
```

Comment:
I've taken the liberty of adding this property to allow for `\n` and `&#10;` which could be usable when working with multiline labels.

Reply:
Good thinking @MadsBuchmann 👌🏻